src/components/RowOperations/RowOperationsButton.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { MutableRefObject } from 'react'
+import { Dropdown } from 'antd'
+import { WidgetTableMeta } from '../../interfaces/widget'
+import styles from './RowOperationsButton.less'
+import { useDispatch } from 'react-redux'
+import { $do } from '../../actions/actions'
+import { DataItem } from '@tesler-ui/schema'
+import RowOperationsMenu from './RowOperationsMenu'
+import { useRowMenuInstance } from '../../hooks/useRowMenu'
+
+/**
+ * {@link RowOperationsMenu | RowOperationsMenu} properties
+ */
+interface RowOperationsButtonProps {
+    /**
+     * Widget meta description
+     */
+    meta: WidgetTableMeta
+    /**
+     * Use when business component differs from widget's (e.g. hierarchies nested level)
+     */
+    bcName?: string
+    /**
+     * Common parent for {@link Table | Table} and {@link RowOperationsButton | RowOperationsButton}
+     */
+    parent: MutableRefObject<HTMLElement>
+}
+
+/**
+ * {@link RowOperationsButton | RowOperationsButton} instance
+ */
+export interface RowOperationsButtonInstance {
+    setRow: (record: DataItem, e: React.MouseEvent<HTMLElement>) => void
+    containerRef: MutableRefObject<HTMLElement>
+}
+
+/**
+ * Button that shows available operations for the row
+ *
+ * On click fetches row meta for clicked row by dispatching
+ * {@link ActionPayloadTypes.bcSelectRecord | bcSelectRecord} and shows {@link RowOperationsMenu | RowOperationsMenu}
+ *
+ * {@link RowOperationsButtonInstance.setRow | setRow} can be used to dynamically place the button
+ * next to hovered row.
+ *
+ * @param props
+ * @param ref - Assigned reference will receive component instance
+ */
+export const RowOperationsButton: React.ForwardRefRenderFunction<RowOperationsButtonInstance, RowOperationsButtonProps> = (
+    { meta, parent, ...props }: RowOperationsButtonProps,
```

Comment:
I don't see any other props except `meta`, `parent`, and `bcName`.
May it be replaced with ```{ meta, parent, bcName }: RowOperationsButtonProps```?

Reply:
This will create naming collision as bcName inside the component is calculated from props.bcName and meta.bcName

Reply:
`const {bcName: propsBcName}: RowOperationsButtonProps`?
But it doesn't matter here. I've got it.